samples/client/petstore/elixir/lib/openapi_petstore/api/fake.ex

```diff
@@ -222,7 +222,7 @@ defmodule OpenapiPetstore.Api.Fake do
   - `{:ok, String.t}` on success
   - `{:error, Tesla.Env.t}` on failure
   """
-  @spec fake_outer_string_serialize(Tesla.Env.client, keyword()) :: {:ok, String.t} | {:error, Tesla.Env.t}
+  @spec fake_outer_string_serialize(Tesla.Env.client, keyword()) :: {:ok, String} | {:error, Tesla.Env.t}
```

Comment:
`String` should be `String.t`

https://hexdocs.pm/elixir/1.15.0/typespecs.html#remote-types